package.json

```diff
@@ -22,7 +22,11 @@
   "author": "Angular2-Meteor (https://github.com/Urigo/angular2-meteor)",
   "license": "MIT",
   "scripts": {
-    "build": "gulp build",
+    "typings": "typings i",
+    "prebuild": "npm run typings && npm run lint",
+    "build": "npm run build-only",
+    "build-only": "tsc",
```

Comment:
@kamilkisiela ok, for https://github.com/Urigo/angular2-meteor-auto-bootstrap there were warnings about `meteor-promise`. `npm build` fails which is bad, and one doesn't sees the reason. Since these warnings are not fixable currently, it'd be good to have better output control. That what I meant.